vmd.py

```diff
@@ -12,18 +12,27 @@
 
 class VMD:
     RESET_FN_NAME = 'reset'
-    def __init__(self, yaml_path) -> None:
-        logging.basicConfig(level=logging.DEBUG)
-
-        vmd_params = load_yaml(yaml_path)
+    UPDATE_FN_NAME = 'update'
```

Comment:
After Sharon gonna create the 2 interfaces the resetable & updatable.
RESET_FN_NAME & UPDATE_FN_NAME need to be delteted.